pom.xml

```diff
@@ -56,12 +56,21 @@
   </pluginRepositories>
 
   <dependencies>
-
+    <dependency>
+      <groupId>ome</groupId>
+      <artifactId>formats-api</artifactId>
+      <version>6.10.0</version>
+    </dependency>
     <dependency>
       <groupId>com.bc.zarr</groupId>
       <artifactId>jzarr</artifactId>
       <version>0.3.4</version>
     </dependency>
+    <dependency>
+      <groupId>org.objenesis</groupId>
+      <artifactId>objenesis</artifactId>
+      <version>2.5.1</version>
```

Comment:
This is the version shipped in `ome-common` (via `kryo` 4.0.2). From `mvn dependency:tree -Dverbose` it looks like it is conflicting with the version from `org.mockito:mockito-inline:3.7.7`.

Declaring it in the POM should suffice although I assume an alternative would be to add and `exclude` in the `mockito` dependency?

Reply:
From a quick discussion with @dgault and @jburel, a possible simpler and more effective option would simply be to declare `mockito` and `testng` as `test` dependencies (via `scope`). This might suffice to exclude these from the openmicroscopy Ivy resolution and prevent the copying of unnecessary JARs into the `lib` folder